src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/BaseShapeTagHelper.cs

```diff
@@ -36,10 +42,14 @@ protected BaseShapeTagHelper(IShapeFactory shapeFactory, IDisplayHelper displayH
         public override async Task ProcessAsync(TagHelperContext tagHelperContext, TagHelperOutput output)
         {
             // Extract all attributes from the tag helper to
-            var properties = output.Attributes
-                .Where(x => !InternalProperties.Contains(x.Name))
-                .ToDictionary(x => LowerKebabToPascalCase(x.Name), x => (object)x.Value.ToString())
-                ;
+            var properties = new Dictionary<string, string>();
+            foreach (var pair in output.Attributes)
```

Comment:
i think it needs to be the following to target the right `Arguments.From(properties)` method

    var properties = new Dictionary<string, object>();